### PR TITLE
Updated LDS projects

### PR DESCRIPTION
**Pull request by ineiti**

### CODE REVIEW

File: data/LDS/projects.yaml [hunk 1]
```diff
@@ -165,7 +177,11 @@ projects:
     information:
       - type: Paper
         title: "Drynx: Decentralized, Secure, Verifiable System for Statistical Queries and Machine Learning on Distributed Datasets"
-        url: https://arxiv.org/pdf/1902.03785.pdf
+        url: https://infoscience.epfl.ch/record/277406?ln=en
+        notes:
+          - label: Published in
+            text: "Ieee Transactions On Information Forensics And Security"
```
Comment (cgrigis) [inline data/LDS/projects.yaml:183]:
```suggestion
            text: "IEEE Transactions On Information Forensics And Security"
```

File: data/LDS/projects.yaml [hunk 2]
```diff
@@ -65,9 +65,17 @@ projects:
...
+            text: "Proceedings on Privacy Enhancing Technologies (PoPETs)"
+            url: https://sciendo.com/article/10.2478/popets-2020-0061
       - type: Paper
         title: "PriFi: A Low-Latency and Tracking-Resistant Protocol for Local-Area Anonymous Communication"
-        url: http://www.cs.yale.edu/homes/jf/PriFi-WPES2016.pdf
+        url: https://infoscience.epfl.ch/record/223389?ln=en
+        notes:
+          - label: Published at
+            text: "ACM Workshop on Privacy in the Electronic Society (WPES), Vienna, Austria, October 24, 2016"
+            url: http://www.cs.yale.edu/homes/jf/PriFi-WPES2016.pdf
```
Comment (cgrigis) [inline data/LDS/projects.yaml:78]: If I am not mistaken, this is on the homepage of one of the paper contributors.
The conference URL seems to be:
    http://wpes2016.di.unimi.it/program

File: data/LDS/projects.yaml [hunk 3]
```diff
@@ -93,11 +101,15 @@ projects:
...
+      date_last_commit: 2021-03-26
     information:
       - type: Paper
         title: "UnLynx: A Decentralized System for Privacy-Conscious Data Sharing"
-        url: https://www.degruyter.com/downloadpdf/j/popets.2017.2017.issue-4/popets-2017-0047/popets-2017-0047.pdf
+        url: https://infoscience.epfl.ch/record/229308?ln=en
+        notes:
+          - label: Published at
+            text: "Privacy Enhancing Technologies Symposium, Minneapolis, MN, USA, July 18–21, 2017"
+            url: https://www.degruyter.com/downloadpdf/j/popets.2017.2017.issue-4/popets-2017-0047/popets-2017-0047.pdf
```
Comment (cgrigis) [inline data/LDS/projects.yaml:112]: The conference site is at: https://petsymposium.org/2017/program.php

**Review by cgrigis — CHANGES_REQUESTED** [review]
A few minor points

File: data/LDS/projects.yaml [hunk 4]
```diff
@@ -65,9 +65,17 @@ projects:
       - type: Paper
         title: "PriFi: Low-Latency Metadata Protection for Organizational Networks"
         url: https://arxiv.org/pdf/1710.10237.pdf
+        notes:
+          - label: Published at
```
Comment (tharvik) [inline data/LDS/projects.yaml:69]:
```suggestion
          - label: Published in
```

File: data/LDS/projects.yaml [hunk 5]
```diff
@@ -65,9 +65,17 @@ projects:
...
+            text: "Proceedings on Privacy Enhancing Technologies (PoPETs)"
+            url: https://sciendo.com/article/10.2478/popets-2020-0061
       - type: Paper
         title: "PriFi: A Low-Latency and Tracking-Resistant Protocol for Local-Area Anonymous Communication"
-        url: http://www.cs.yale.edu/homes/jf/PriFi-WPES2016.pdf
+        url: https://infoscience.epfl.ch/record/223389?ln=en
```
Comment (tharvik) [inline data/LDS/projects.yaml:74]: no need to force language of the page.

```suggestion
        url: https://infoscience.epfl.ch/record/223389
```

File: data/LDS/projects.yaml [hunk 6]
```diff
@@ -93,11 +101,15 @@ projects:
     code:
       type: Lab GitHub
       url: https://github.com/ldsec/unlynx
-      date_last_commit: 2020-03-19
+      date_last_commit: 2021-03-26
     information:
       - type: Paper
         title: "UnLynx: A Decentralized System for Privacy-Conscious Data Sharing"
-        url: https://www.degruyter.com/downloadpdf/j/popets.2017.2017.issue-4/popets-2017-0047/popets-2017-0047.pdf
+        url: https://infoscience.epfl.ch/record/229308?ln=en
```
Comment (tharvik) [inline data/LDS/projects.yaml:108]:
```suggestion
        url: https://infoscience.epfl.ch/record/229308
```

File: data/LDS/projects.yaml [hunk 7]
```diff
@@ -165,7 +177,11 @@ projects:
     information:
       - type: Paper
         title: "Drynx: Decentralized, Secure, Verifiable System for Statistical Queries and Machine Learning on Distributed Datasets"
-        url: https://arxiv.org/pdf/1902.03785.pdf
+        url: https://infoscience.epfl.ch/record/277406?ln=en
```
Comment (tharvik) [inline data/LDS/projects.yaml:180]:
```suggestion
        url: https://infoscience.epfl.ch/record/277406
```

**Review by cgrigis — APPROVED** [review]

**Pull request merged**